zebrad/src/components/sync.rs

```diff
@@ -116,6 +116,14 @@ pub(super) const BLOCK_DOWNLOAD_TIMEOUT: Duration = Duration::from_secs(15);
 /// failure loop.
 pub(super) const BLOCK_VERIFY_TIMEOUT: Duration = Duration::from_secs(180);
 
+/// Controls how long we wait for a transaction download request to complete.
+/// TODO: review value and rationale
+pub(super) const TRANSACTION_DOWNLOAD_TIMEOUT: Duration = Duration::from_secs(15);
```

Comment:
This is probably fine for the current low transaction volume on the Zcash network.

Unlike syncer downloads, a full transaction queue drops hashes, and transaction verifications are independent of each other. (But they might not stay independent, see #2682.)
So those parts of the `BLOCK_DOWNLOAD_TIMEOUT` rationale might not apply.

But I still think it's good to be consistent, unless there's a strong reason to differ.

Reply:
I've moved the constants to `downloads.rs` (it doesn't make much sense for them to be here) and added some simple docs in a68bf1c4b041.